src/services/anchor-service.ts

```diff
@@ -326,6 +326,10 @@ export default class AnchorService {
         throw new Error(`No valid ceramic document found with docId ${docId.toString()}`)
       }
 
+      if (doc.tip.toString() != request.cid) {
+        logger.warn(`When loading document ${doc.id.toString()} at commit ${request.cid.toString()}, doc was returned at tip ${doc.tip.toString()}`)
```

Comment:
Then why do we do toString here then?

Reply:
this is probably the part that should change I guess

Reply:
Okay, updated to remove the toString here